metis/bin/metis_client

```diff
@@ -170,6 +170,14 @@ class MetisClient
       ]
     )
 
+    if response.code == '503'
+      sleep 60
```

Comment:
Minor nitpick, could we log that the program is sleep?  Might come off as a stuck program, especially if I'm uploading a small file at the time and suddenly I'm waiting a minute for any output changes.